pkg/runtime/adoption_reconciler.go

```diff
@@ -466,3 +478,26 @@ func NewAdoptionReconciler(
 		},
 	}
 }
+
+// LogAdoptedResource returns a logger with log values set for the resource's kind,
+// namespace, name, etc
+func LogAdoptedResource(
```

Comment:
Can you place this into `pkg/runtime/log` and call it `AdaptAdoptedResource` and follow the same pattern used in there for a `DebugAdoptedResource` and `InfoAdoptedResource` function please?

https://github.com/aws-controllers-k8s/runtime/blob/10d1b5339b6030fcaedf65d84e637d12a8779ab1/pkg/runtime/log/resource.go#L22-L69